war/src/main/less/pluginSetupWizard.less

```diff
@@ -544,12 +552,24 @@
 
         .dependent {
             color: #aaa;
-            padding-left: .4em;
+            padding-left: 10px;
 
             &:before {
                 content: ' ** ';
             }
         }
+        
+        .dependency-legend {
+            color: #aaa;
```

Comment:
We can change the dependency color and make them match, but it is confusing when it's a different color, when the dependencies are already a different color. @daniel-beck